app/controllers/content/casebooks_controller.rb

```diff
@@ -8,7 +8,8 @@ class Content::CasebooksController < Content::NodeController
   def new
     @casebook = Content::Casebook.create(public: false, collaborators: [Content::Collaborator.new(user: current_user, role: 'owner')])
     logger.debug @casebook.errors.inspect
-    redirect_to edit_casebook_path @casebook
```

Comment:
originally this redirected to edit_casebook_path which then redirected to layout_casebook_path